src/Module/Services/ItemFiltersService.ts

```diff
@@ -0,0 +1,71 @@
+import {IItemFilterSet} from '@src/Types/IItemFilterSet';
+import {IItemSchema} from '@src/Schema/IItemSchema';
+import data from '@src/Data/Data';
+
+export class ItemFiltersService
+{
+	public filter: IItemFilterSet;
+	public items: IItemSchema[] = Object.values(data.getAllItems());
+
+	private defaultFilterState: IItemFilterSet = {
+		showAdvanced: false,
```

Comment:
we could do this using bootstrap's [collapse](https://getbootstrap.com/docs/4.5/components/collapse/), has a fancy animation included as well.

Reply:
Sure, but if we do, then it will collapse filters once switching between states, because of how it's done for now